requirements.txt

```diff
@@ -4,7 +4,7 @@ boto3==1.20.45
 botocore==1.23.45
 click==8.0.3
 requests==2.27.1
-ruamel.yaml==0.17.20
+PyYAML==6.0
```

Comment:
Any dedicated reasons for using pyyaml instead of ruamel.yaml here?

Reply:
yes, I found it easier to read YAML using PyYAML vs multiple steps needed in `ruamel.yaml`.

Reply:
If the ruamel.yaml has passed our rpm process, I'd recommend to keep it except some mandatory reason to use. I'm not sure the lib change will break the rpm.

Reply:
I verified and it works fine because we also have python3-pyyaml available.

Reply:
Ok and agree to merge. Please have a test on our testing environment to see if any problems.